tlklan/index.php

```diff
@@ -10,9 +10,10 @@
 }
 
 // Enable error reporting and display errors eventually
-error_reporting(E_ALL);
+error_reporting(E_ERROR);
 
 if (getenv('APP_ENV') === 'local') {
+    error_reporting(E_ALL);
```

Comment:
Whitespace

Reply:
Can't be helped, editor will not produce tabs ;) just 4 spaces on tab as it should be.

Reply:
On a more serious note, maybe we should convert the rest of the code to using spaces aswell, for PSR reasons and such...